### PR TITLE
Move 1/16.0f to packing routine

### PR DESCRIPTION
**Pull request by digantdesai**
before:
    fp_acc = 1/16 * (vksum * 16 + float(int_acc * 16) * scale)

after:
    fp_acc = vksum + float(int_acc * 16) * scale / 16

### CODE REVIEW

File: src/microparams-init.c [hunk 1]
```diff
@@ -951,7 +951,8 @@ void xnn_init_blockwise_scale_fp32_params(
       const size_t tile_size = channels_tile;
       for (size_t tile_offset = 0; tile_offset < tile_size; tile_offset++) {
         size_t scale_index = (tile_start + tile_offset) * num_blocks + block_start;
-        unaligned_indexed_store_f32(packed_w, tile_offset, scale[scale_index]);
+        // 1/16 because the weight are << 4 in the innermost loop to save a shift
+        unaligned_indexed_store_f32(packed_w, tile_offset, scale[scale_index]/16.0f);
```
Comment (fbarchard) [inline src/microparams-init.c:955]: unaligned_indexed_store_f32(packed_w, tile_offset, scale[scale_index] * (1.0f / 16.0f));
Note I plan to change 4 bit to low bits for qd8

File: bench/gemm-benchmark.h [hunk 2]
```diff
@@ -402,7 +402,7 @@ void GEMMBenchmark(benchmark::State& state,
 
   const size_t mc = state.range(0);
   const size_t nc = state.range(1);
-  const size_t kc = state.range(2);
+  const size_t kc = round_up_po2(state.range(2), 2 * kr * sr);
```
Comment (fbarchard) [inline bench/gemm-benchmark.h:405]: kc does not require alignment?

File: src/microparams-init.c (hunk 1 above)
Comment (digantdesai) [reply src/microparams-init.c:955]: Yep, `fdiv` should be avoided here. Compilers aren't smart enough.

File: bench/gemm-benchmark.h (hunk 2 above)
Comment (digantdesai) [reply bench/gemm-benchmark.h:405]: This is to satisfy the constraints kernels have for kc % bl == 0 i.e. padding for benchmark to support various shapes. Not sure if I follow your comment about alignment.

File: src/microparams-init.c (hunk 1 above)
Comment (digantdesai) [reply src/microparams-init.c:955]: cc @GregoryComer  - he is taking over this PR.

**Pull request closed without merging**